packs/data/extinction-curse-bestiary.db/death-drider.json

```diff
@@ -102,7 +199,7 @@
                 },
                 "deathNote": true,
                 "description": {
-                    "value": "<p><strong>Trigger</strong> The death drider is reduced to 0 Hit Points</p>\n<hr />\n<p><strong>Effect</strong> The death drider explodes in a shower of spiders. Each adjacent creature must succeed at a @Check[type:reflex|dc:31|traits:damaging-effect] save or take [[/r 3d6[piercing]]] damage and be exposed to death drider venom.</p>"
+                    "value": "<p><strong>Trigger</strong> The death drider is reduced to 0 Hit Points</p>\n<hr />\n<p><strong>Effect</strong> The death drider explodes in a shower of spiders. Each adjacent creature must succeed at a @Check[type:reflex|dc:31|traits:damaging-effect] save or take [[/r 3d6[piercing]]]{3d6 piercing damage} and be exposed to death drider venom.</p>"
```

Comment:
These damage labels should generally be left out due to Shark's changes.

Reply:
I will go back and revise tomorrow